capa/v25.0.0/README.md

```diff
@@ -0,0 +1,162 @@
+
+- Flat DNS structure - Bringing more flexibility into workload cluster management across regions. The flat DNS structure will not include the Management Cluster name anymore, but more generic DNS domain prefixed with cluster ID. This provides advantage where the Workload Cluster is no longer tied to the region nor Management Cluster, extending possibilities for any future migrations.  Old and new DNS are both available during and after migration, while new DNS needs to be used after migration. We will synchronize with customers for the old DNS phase out after the migrations per management cluster.
+- Only public DNS zone is supported - Private hosted zones are no longer created. However, both zones (public and private) will be available during and after the migration. We will eventually work together to remove the private zones with each customer. Ingresses that depend on the cluster base domain must be updated after the migration.
+- Cluster apps moved to `org-` namespaces - This change will allow to simplify RBAC and enable GitOps managed clusters to be created with a pre-defined set of applications.
+- One set of controllers - On contrary to Vintage, where each release had a set of independent operators, the CAPA solution will manage all releases with a single set of controllers. This will result in fewer components running on the management clusters, hence lowering costs of operations as well as speeding up any required hotfixes.
+- `GP2` volumes not supported - The majority of customers is already using `gp3` volumes, as we are refreshing the infrastructure, deprecated `kubernetes.io/aws-ebs` provisioner creating `gp2` volumes will also be removed
+- Node pool defaults are changed - With CAPA by default, if not explicitly configured, node pools will now share a single subnet. Migrated Vintage clusters will not be affected. With that change the node limit will be determined by cluster VPC size, where additional VPC CIDRs can be added. On the contrary to Vintage, CAPA also allows to configure and deploy node pools together with the cluster object by default.
+- [Teleport](https://docs.giantswarm.io/vintage/platform-overview/security/cluster-security/cluster-access/#admin-access-via-teleport) - Giant Swarm will now utilize `Teleport` for Kubernetes API and direct node access. It helps to strengthen security and simplifies compliance with regulations and network topology. This service is only available to Giant Swarm engineers, while customers can obtain audit logs for any operations and logins performed by Giant Swarm support.
+
+## Vintage to CAPA Migration
+
+The migration itself is a fully automated process ran by Giant Swarm engineers using a `migration-cli` that handles all infrastructure as well as workload migrations. 
+*The experience in the migration process should be the same as in an usual upgrade of the workload cluster, where the nodes rollout takes place.*
```

Comment:
This is not true for customers using GitOps because they have to migrate their configuration to use the new API (CRs).

Reply:
i can add disclaimer for that

Reply:
adjusted